src/Components/Artwork/FlatGridItem.tsx

```diff
@@ -59,6 +59,8 @@ const FlatGridItem: React.FC<FlatGridItemProps> = ({ artwork, onClick }) => {
   const shouldRenderProgressBar =
     isAuctionArtwork && isExtendedBiddingEnabledSale && !hasEnded
 
+  const enablePrefetch = sale?.isClosed ?? true
```

Comment:
Could an upcoming sale be classified as closed? Thinking that the  most important prefetches are those for auctions that users can still register for, in order to increase engagement.

Reply:
Nope, upcoming sales aren't closed, so this won't prefetch those either. This will just skip prefetching when the sale _has_ closed.